lib/frodo/profile.rb

```diff
@@ -2,16 +2,32 @@
 
 module Frodo
   class Profile
-    def self.get(id:, token:, version: 'latest')
-      gandalf_url = ENV['GANDALF_URL'].to_s + "/profiles/#{id}/#{version}"
-      new(gandalf_url: gandalf_url, token: token)
+    def self.get(id:, version: 'latest')
+      gandalf_url = ENV['GANDALF_URL'].to_s + "/profiles/#{id}?version=#{version}"
```

Comment:
Is there ever a chance that our environment variable will not be a string? Is to protect against nils?

Reply:
yeah if nil you get undefined method +